test/parallel/test-stream-base-prototype-accessors.js

```diff
@@ -10,9 +10,9 @@ const assert = require('assert');
 // Or anything that calls StreamBase::AddMethods when setting up its prototype
 const TTY = process.binding('tty_wrap').TTY;
 
-// Should throw instead of raise assertions
 {
-  const msg = /TypeError: Method \w+ called on incompatible receiver/;
+  // Should throw instead of raise assertions
+  const msg = /TypeError: Illegal invocation/;
```

Comment:
Even better, change all uses of `msg` below to `TypeError`, because the exact error message may differ depending on the JavaScript engine.

Reply:
👍 Fixed.